ModelBundle/Repository/ContentRepository.php

```diff
@@ -405,6 +424,23 @@ public function countByContentTypeInLastVersion($contentType = null)
     }
 
     /**
+     * {@inheritDoc}
+     * @see \OpenOrchestra\ModelInterface\Repository\ContentRepositoryInterface::countByContentTypeAndSiteInLastVersion()
+     */
+    public function countByContentTypeAndSiteInLastVersion($contentType = null, $siteId = null)
```

Comment:
The method is named countByContentTypeAndSite but the contentType and the siteId can be omitted, is it normal ? Maybe you could remove 'ContentTypeAndSite' from the name or remove the default values.